README.md

```diff
@@ -1,28 +1,92 @@
+>
+> Not quite.
+> There are lots of ways that open source is amazing, but it doesn’t exist outside the laws of physics. You have to put work in, to get work out.
+>
+>You only get contributions after you have put in a lot of work.
+>You only get contributions after you have users.
+>You only get contributions after you have documentation.”
+>
+From @ericholscher [guide to writing docs](https://www.writethedocs.org/guide/writing/beginners-guide-to-docs/)
 
-All doc-related issues should be [submitted to here](https://github.com/jina-ai/jina/issues/new).
+## Table of contents
+*  [Hierarchical structure](#Hierarchical)
+* [Missing content](#Missing)
+*  [How to add pages](#howto)
+*  [Jina AI style guide](#style)
+* [Updating Docstrings](#docstrings)
+* [Updating API](#apis)
+* [Technical aspects of this site](#tech)
 
-## Read the Docs Locally
+## Hierarchical structure <a name="Hierarchical"></a>
 
-You can read the documentations locally via:
+Jina docs adheres to the following hierarchical structure. Each Jina product has its own section, containing three subsections below.
+|  |  |
+|--|--|
+| **Overview** |A high level conceptual overview of the product. Introducing terms and broad architectural concepts. Content here should apply to all Jina users. For example, all users should understand what a Pod is, but only some users need to understand deployment on a GPU.
+ |**Developer Guides**  | Are technical how-to guides/tutorials which describe product features or implementations. Assumes basic knowledge of the product and related terms. |
+| **API References**| Are detailed descriptions of the product API. Possibility auto-generated from docstrings or open API references. Describes how the methods work and which parameters can be used.  |
 
-```bash
-# Clone the code.
-git clone https://github.com/jina-ai/docs.git
 
-# Install dependencies.
-pip install -r requirements.txt
+## Missing content <a name="Missing"></a>
 
-# Clean & build docs locally
-make clean
-make html
+If you find a gap in our documentation. Please submit a Github issue [here](https://github.com/jina-ai/docs).
 
-# Serve the docs website with Python 3
-python -m http.server 8080 -d _build/html
-```
+## How to add pages <a name="howto"></a>
+
+For getting started pages and developer guides:
+
+1.  A page can be written in Markdown or reStructuredText formats.
+
+2.  Use a template from the [page_templates](https://github.com/jina-ai/docs/tree/master/page_templates) folder.
+
+3.  Use git to clone the docs repo.
+
+4.  Create a git new branch
+
+5.  Add your file to the chapters folder.
+
+8.  Push your branch and create a pull request.
+
+9.  Add at least two people to your pull request review. One product manager and one developer.
+
+10.  You can preview how the docs website will look with your changes. Inside your pull request, navigate to checks and click preview with netlify.
```

Comment:
Wrap "preview with netlify" in quotes. Check correct capitalization for that text